Rover/GCS_Mavlink.cpp

```diff
@@ -1206,3 +1273,6 @@ uint8_t GCS_MAVLINK_Rover::send_available_mode(uint8_t index) const
 
     return mode_count;
 }
+
+#if AP_RANGEFINDER_ENABLED
+#endif
```

Comment:
Left over from a rebase?

Reply:
Not in this case - I moved the send-water-depth-code to be next to the send-rangefinder-code as the `#if`s already existed and the methods are related....

Removed these lines.